### ISSUE TITLE
Large scene with 8000 images

### ISSUE THREAD

**Issue by zyc-bit** [dcharatan/flowmap#53]
Hello everyone, I have a video shot in a large indoor scene, which has generated over 8,000 images, extracted in sequence from the video. I'm wondering if current systems support processing such a large number of images? It seems like the GPU memory might not be sufficient.

**Comment by linwk20** [comment]
Maybe mini-batch optimization will help? currently the code using full-batch so is not scalable.

**Comment by zyc-bit** [comment]
Oh, okay. Maybe this is a direction worth trying. Have you also been researching this project recently?


> Maybe mini-batch optimization + gradient accumulation will help? currently the code using full-batch so is not scalable.

**Comment by linwk20** [comment]
no, but it is easy to try, let me know if it works, you can try this on official small dataset first to see if accuracy degrade.
